libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -23,18 +23,23 @@ logScope:
   topic = "PubSubPeer"
 
 type
-    PubSubPeer* = ref object of RootObj
-      proto: string # the protocol that this peer joined from
-      sendConn: Connection
-      peerInfo*: PeerInfo
-      handler*: RPCHandler
-      topics*: seq[string]
-      sentRpcCache: TimedCache[string] # cache for already sent messages
-      recvdRpcCache: TimedCache[string] # cache for already received messages
-      refs*: int # refcount of the connections this peer is handling
-      onConnect: AsyncEvent
-
-    RPCHandler* = proc(peer: PubSubPeer, msg: seq[RPCMsg]): Future[void] {.gcsafe.}
+  PubSubObserver* = ref object
+    onRecv*: proc(peer: PubSubPeer; msgs: var RPCMsg) {.gcsafe.}
```

Comment:
Minor style issue, I've seen the use `;` but, IMO I don't think we use this style anywhere, for consistency can we stick with `,` instead (unless there is a legit use case)?

Reply:
Well actually it's a better habit.
You can aggregate names on the same types with `;` and avoid confusion.
Example: `a, b: int; c: float`, with comma the compiler has issues (at least in the past) and overall confusing.

But I can fix, it's just a old habit.